Styles/header.css

```diff
@@ -0,0 +1,53 @@
+.nav-bar {
+  background: transparent;
+  color: var(--text-color);
+  padding: 0;
+}
+
+.nav-logo {
+  margin: 1rem;
+}
+
+.nav-search {
+  min-width: 30%;
```

Comment:
I think it should be atleast 80-90% as the X symbol will show half way in the UI